src/main/java/com/meilisearch/sdk/KeysHandler.java

```diff
@@ -29,8 +31,7 @@ public class KeysHandler {
      * @throws MeilisearchException if client request causes an error
      */
     Key getKey(String uid) throws MeilisearchException {
-        String urlPath = "/keys/" + uid;
-        return httpClient.get(urlPath, Key.class);
+        return httpClient.get(new KeysQuery().toQuery(uid), Key.class);
```

Comment:
>I'm not sure if this is the best way to do it

How would you do it?
>should getIndex work the same way or should getKey work like getIndex?

Why should it work the same way?
I created a `toQuery()` method in `Key` class because this piece of code is repeated twice.